dist/package.sh

```diff
@@ -83,11 +83,11 @@ for target in "${targets[@]}"; do
     else
         install -D "node_modules/$target" "$DIR/node_modules/$target"
     fi
-    sed -i "/sourceMappingURL/d" "$DIR/node_modules/$target"
+    sed -i.bak "/sourceMappingURL/d" "$DIR/node_modules/$target"
```

Comment:
```suggestion
    sed -i"" "/sourceMappingURL/d" "$DIR/node_modules/$target"
```

Reply:
afaik `sed -i""` doesn't work on BSD/macOS. But unless we can safely delete all `.bak` files in `
"$DIR/node_modules/$target"`, it might be better to let everyone using macos (which is currently only me anyways) install gnu-sed.

Reply:
Tested and works fine on Mac OS

Reply:
Actually... no, it creates a file suffixed with `-e`. :slightly_frowning_face: